src/ruby/end2end/grpc_class_init_client.rb

```diff
@@ -80,6 +80,33 @@ def get_test_proc(grpc_class)
     return proc do
       GRPC::Core::ChannelCredentials.new
     end
+  when 'xds_channel_credentials'
+    return proc do
+      GRPC::Core::XdsChannelCredentials.new(GRPC::Core::ChannelCredentials.new)
+    end
+  when 'server_credentials'
+    return proc do
+      test_root = File.join(File.dirname(__FILE__), '..', 'spec', 'testdata')
+      GRPC.logger.info("test root: #{test_root}")
```

Comment:
nit: can we please remove this log line here, or just have it use `STDERR.puts` ?

Reason for this is this test is meant to exercise the case where we load `GRPC::Core::ServerCredentials`  as the first thing we do in the process, before using any other grpc library APIs

Reply:
deleted